apps/src/templates/CodeWorkspace.jsx

```diff
@@ -44,7 +49,12 @@ class CodeWorkspace extends React.Component {
     Object.keys(nextProps).forEach(
       function(key) {
         // isRunning and style only affect style, and can be updated
```

Comment:
nit: can you update this comment to explain the new keys that have been added?

Reply:
Yes -  good catch!